tools/system_libs.py

```diff
@@ -1800,6 +1803,11 @@ class libmimalloc(MTLibrary):
     '-DMI_MALLOC_OVERRIDE',
     # TODO: add build modes that include debug checks 1,2,3
     '-DMI_DEBUG=0',
+    # disable `assert()` in emmalloc
+    '-DNDEBUG',
+    # avoid use of `__builtin_thread_pointer()`
+    # FIXME: https://github.com/llvm/llvm-project/issues/117433
```

Comment:
I'm not sure we want a FIXME here since we are using musl as our libc so this seems correct in any case.

Reply:
a090ba89182812faed943b864fff8b7046a8e6fa